pyproject.toml

```diff
@@ -1,8 +1,86 @@
 [build-system]
-# AVOID CHANGING REQUIRES: IT WILL BE UPDATED BY PYSCAFFOLD!
-requires = ["setuptools>=46.1.0", "setuptools_scm[toml]>=5", "wheel"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling"]
+build-backend = "hatchling.build"
 
-[tool.setuptools_scm]
-# See configuration details in https://github.com/pypa/setuptools_scm
-version_scheme = "no-guess-dev"
+[project]
+name = "SALib"
+dynamic = ["version"]
+description = "Tools for sensitivity analysis. Contains Sobol', Morris, and FAST methods"
```

Comment:
Update the description with the complete list of methods provided. Add 'global' in front of 'sensitivity analysis'